x-pack/packages/kbn-elastic-assistant/impl/alerts/settings/alerts_settings.test.tsx

```diff
@@ -19,8 +19,8 @@ describe('AlertsSettings', () => {
 
   it('updates the knowledgeBase settings when the switch is toggled', () => {
     const knowledgeBase: KnowledgeBaseConfig = {
-      alerts: false,
-      assistantLangChain: false,
+      isEnabledRAGAlerts: false,
+      isEnabledKnowledgeBase: false,
```

Comment:
nit: consider swapping the subject and verb such that the variable reads like a sentence when used in a expression, i.e.
`isEnabledRAGAlerts` --> `isRagAlertsEnabled`
`isEnabledKnowledgeBase` --> `isKnowledgeBaseEnabled`